src/commands/help.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from discord import Color, Embed, Interaction, app_commands
+from discord.ext import commands
+
+if TYPE_CHECKING:
+    from src.main import AeBot
+
+
+class HelpCog(commands.GroupCog, group_name="help"):
```

Comment:
```suggestion
class HelpCog(commands.Cog):
```
C'est redondant, de devoir écrire /help help, plutôt que /help